wwwroot/inc/interface-config.php

```diff
@@ -907,6 +910,14 @@ function renderTagRowForEditor ($taginfo, $parent_name = NULL, $level = 0)
 	$sparams = array ('name' => 'parent_id', 'id' => 'nodeid_' . $taginfo['id'], 'class' => 'nodelist-popup');
 	echo getSelect ($poptions, $sparams, $taginfo['parent_id'], FALSE);
 
+	if($taginfo['is_assignable'] == 'yes')
+	{
+		$class = getObjectClass('tag', $taginfo);
+		echo "</td><td class='${class}'>".getColorSelect("colorid_".$taginfo['id'], $taginfo['color']).'</td>';
+	}
+	else
+		echo "<td></td>";
+
```

Comment:
This needs to be `<td>&nbsp;</td>` to keep CSS working in all browsers (at least it used to be so and all the other table printing code does it this way).